rice-datanode/include/ClientDatanodeProtocolImpl.h

```diff
@@ -30,18 +32,18 @@ class ClientDatanodeTranslator {
 
   int getPort();
   RPCServer getRPCServer();
+
  private:
   std::string Serialize(google::protobuf::Message &);
   void InitServer();
   void RegisterClientRPCHandlers();
   void Config();
   void logMessage(google::protobuf::Message &req, std::string req_name);
 
-  FsServerDefaultsProto server_defaults;
+  hadoop::hdfs::FsServerDefaultsProto server_defaults;
```

Comment:
this works, but what i've been doing is changing:
`using namespace hadoop::hdfs`
to
`using hadoop::hdfs::FsServerDefaultsProto `
(and copying that for each specific import)
which is i think what the lint error is suggesting. what do you think is better?

Reply:
I have no preference.

I'll let you pick one and we'll standardize on that for the rest of the codebase.

Reply:
let's do using-declarations so an import for each. hopefully that will keep things clearer for everyone going forward

Reply:
@anant12